src/main/twirl/uk/gov/hmrc/hmrcfrontend/views/components/hmrcAddToAList.scala.html

```diff
@@ -0,0 +1,104 @@
+@*
```

Comment:
just realised that I forgot to fix the indentation on this file. Intellij defaulted to 4 spaces for twirl templates and I never remembered to adjust the setting, I'll push an update that reformats this file after lunch 👍